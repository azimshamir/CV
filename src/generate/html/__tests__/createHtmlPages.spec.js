import createHtmlPages from "../createHtmlPages";
import markdown from "./markdown.md";

const MARKDOWN_AS_HTML = `<div class="page" id="p1"><h1 id="hello-world">Hello World</h1>
<p>This is some <em>markdown</em>.</p>
<p><strong>FOR REAL!</strong></p>
</div>`

describe("createHtmlPages", () => {
  it("generates correct html", () => {
    expect(createHtmlPages(markdown)).toBe(MARKDOWN_AS_HTML);
  });
});
