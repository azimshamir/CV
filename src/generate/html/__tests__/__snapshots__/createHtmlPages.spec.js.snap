// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`createHtmlPages generates correct html 1`] = `
"<div class=\\"page\\" id=\\"p1\\"><h1 id=\\"hello-world\\">Hello World</h1>
<p>This is some <em>markdown</em>.</p>
<p><strong>FOR REAL!</strong></p>
</div>"
`;
